File: Flutter/sqlite_app/lib/students.dart
class Students {
  final int? id;
  final String code;
  final String name;
  final String dept;
  final String phone;

  Students({
    this.id,
    required this.code,
    required this.name,
    required this.dept,
    required this.phone,
  });

  Students.fromMap(Map<String, dynamic> res)
      : id = res['id'],
        code = res['code'],
        name = res['name'],
        dept = res['dept'],
        phone = res['phone'];

  Map<String, Object?> toMap() {
    return {'id': id, 'code': code, 'name': name, 'dept': dept, 'phone': phone};
  }
}

File: Swift/SQLite/SQLite/Students.swift
//
//  Students.swift
//  SQLite
//
//  Created by Jeong Yun Hyeon on 2023/02/19.
//

import Foundation

struct Students {
    var id: Int
    var name: String
    var dept: String
    var phone: String
    
    init(id: Int, name: String, dept: String, phone: String) {
        self.id = id
        self.name = name
        self.dept = dept
        self.phone = phone
    }
}
